SPECS/kernel-mshv/kernel-mshv.spec

```diff
@@ -129,6 +138,20 @@ install -vm 644 vmlinux %{buildroot}%{_libdir}/debug/lib/modules/%{uname_r}/vmli
 # `perf test vmlinux` needs it
 ln -s vmlinux-%{uname_r} %{buildroot}%{_libdir}/debug/lib/modules/%{uname_r}/vmlinux
 
+cat > %{buildroot}/boot/linux-%{uname_r}.cfg << "EOF"
+# GRUB Environment Block
```

Comment:
This grub environment block is not expected to be used in 3.0. Instead, we use grub2-mkconfig to generate the grub.cfg.

Reply:
What is the plan for ingesting/creating a MAP AKS-kata image for this? I imagine we will need to take out some of the manual sed'ing steps in AgentBaker and simply rely on the menuentry template in 50_mariner_mshv_menuentry.

Reply:
discussed with Mitch. This PR was not taking account of the grub2 work that had already happened in 3.0-dev. We'll revisit

Reply:
Rebased against 3.0-dev branch.